draft-ietf-rats-architecture.md

```diff
@@ -419,19 +419,20 @@ environment (after the initial one that is a root-of-trust) has the duty
 of measuring its next environment before it is started. Therefore, creating
 a layered boot sequence and correspondingly enabling Layered Attestation.
 
-## Composite Attester {#compositeattester}
+## Composite Device {#compositedevice}
 
-A Composite Attester is an entity composed of multiple sub-entities such that its
+A Composite Device is an entity composed of multiple sub-entities such that its
 trustworthiness has to be determined by the appraisal of all these sub-entities.
+
 Each sub-entity has at least one Attesting Environment collecting the claims
 from at least one Target Environment, then this sub-entity generates Evidence
 about its trustworthiness. Therefore each sub-entity can be called an Attester.
-Among these Attesters, there may be only some, which can be called Lead Attesters,
-that have the ability to communicate with the Verifier. Other Attesters
-don't have this ability, but they are connected to the Lead Attesters via internal
-links or network connections, and they are appraised by the Lead Attester's.
+Among all the Attesters, there may be only some which have the ability to communicate
+with the Verifier while others do not.
+Those that do not are connected to the lead Attesters via internal
+links (or private network connections), and they are appraised by the lead Attester.
 
-For example, a carrier-grade router is a composite device consisting of a chassis and multiple slots.
+For example, a carrier-grade router is consisted of a chassis and multiple slots.
```

Comment:
grammar: s/is consisted of/consists of/ 
(or revert the change if you prefer the original)